src/ol_infrastructure/infrastructure/mongodb_atlas/__main__.py

```diff
@@ -0,0 +1,120 @@
+import pulumi
+import pulumi_aws as aws
+import pulumi_mongodbatlas as atlas
+
+from bridge.lib.magic_numbers import DEFAULT_MONGODB_PORT
+from ol_infrastructure.lib.aws.ec2_helper import default_egress_args
+from ol_infrastructure.lib.ol_types import AWSBase
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+
+###############
+# STACK SETUP #
+###############
+atlas_config = pulumi.Config("mongodb_atlas")
+env_config = pulumi.Config("environment")
+stack_info = parse_stack()
+network_stack = pulumi.StackReference(f"infrastructure.aws.network.{stack_info.name}")
+consul_stack = pulumi.StackReference(
+    f"infrastructure.consul.{stack_info.env_prefix}.{stack_info.name}"
+)
+vault_stack = pulumi.StackReference(
+    f"infrastructure.vault.operations.{stack_info.name}"
+)
+
+#############
+# VARIABLES #
+#############
+environment_name = f"{stack_info.env_prefix}-{stack_info.env_suffix}"
+target_vpc = network_stack.require_output(env_config.require("target_vpc"))
+business_unit = env_config.get("business_unit") or "operations"
+aws_config = AWSBase(tags={"OU": business_unit, "Environment": environment_name})
+max_disk_size = atlas_config.get("disk_autoscale_max_gb")
+max_instance_type = atlas_config.get("cluster_autoscale_max_size")
+min_instance_type = atlas_config.get("cluster_autoscale_min_size")
```

Comment:
Don't see where this is defined

Reply:
It's not defined for the stacks that I've set up, just exposing it as an option in the code.